docker-compose-test.yml

```diff
@@ -7,15 +7,13 @@ services:
       dockerfile: Dockerfile.test
     links:
       - db
-      - selenium
     environment:
       GEM_HOME: '/rubygems'
       BUNDLE_PATH: '/rubygems'
       DB_HOST: 'db'
       DB_USERNAME: 'postgres'
       RAILS_ENV: 'test'
-      SELENIUM_URL: 'http://selenium:4444/wd/hub'
-      TEST_BROWSER: 'chrome'
+      TEST_BROWSER: 'firefox_local'
```

Comment:
Chrome seems broken at the moment, filling in fields that contain special characters doesnt seem to work